docs/python/programming-guide.md

```diff
@@ -10,8 +10,8 @@ First, without using DBOS, we'll build an app that records greetings to two diff
 Then, we'll add DBOS durable execution to the app in **just four lines of code**.
 Thanks to durable execution, the app will always write to both systems consistently, even if it is interrupted or restarted at any point.
 
-This guide assumes you have a Postgres database running locally.
-If not, see the [quickstart](../quickstart.md) for instructions on how to set it up.
+_**This guide assumes you have a Postgres database running locally and PGPASSWORD is exported in your environment.**_
```

Comment:
I would take out the part about PGPASSWORD-- if they're doing the new local-cloud-database thing then they don't need PGPASSWORD.

Reply:
Maybe we should reword as follows, to be inclusive:

`This guide assumes you have a Postgres database running locally. You might need to export PGPASSWORD in your environment. See the [quickstart](../quickstart.md) for instructions on how to set up Postgres.`

Reply:
Updated. Though we might need to revisit in light of upcoming PG requirements changes?